pkg/defaults/policies/files/pod_exec.json

```
@@ -47,6 +47,15 @@
           "namespace": "openshift-etcd"
         }
       }
+      },
+      {
+      "name": "Don't alert on deployment workspace\\w{16} in namespace openshift-terminal",
```

```suggestion
      "name": "Don't alert on deployments workspace\\w{16} in namespace openshift-terminal; note we assume how the devworkspace operator generates deployment names",
```
an attempt to store as much of the context you shared in the source code.